.eslintrc

```diff
@@ -12,6 +12,10 @@
         "plugin:react/recommended"
     ],
     "rules": {
-        // overrides
+        "no-console": 0
```

Comment:
I'm assuming this allows `console.logs` in the code? Agreed, I would prefer to see it enforced actually